docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1353,6 +1358,28 @@ xrdsst backup restore --ss <SECURITY_SERVER_NAME> --file <BACKUP_FILENAME>
 
 * <SECURITY_SERVER_NAME> name of the security server, e.g. ss1
 * <BACKUP_FILENAME> file name of the backup to be restored from, e.g. conf_backup_20210713-161054.tar
+
+#### 4.2.12 Diagnostics management
+
```

Comment:
In other tasks that are listed, we put the table with a brief description of the properties that are shown, it would be good to add them for a bit of consistency

Reply:
Added example and description of the output